src/contexts/EditableContext.tsx

```diff
@@ -0,0 +1,43 @@
+import { OnDragEndResponder } from "@hello-pangea/dnd"
+import { createContext, PropsWithChildren, useContext } from "react"
+
+interface EditEvent {
+  target: {
+    id: string
+    value?: string
+  }
+}
+
+interface EditableContextReturn {
```

Comment:
sounds like a verb, can make a noun? EditableContextProperties?

Reply:
Actually this is convention because the type is for the return value of `useContext`! 

See [here](https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/editable/src/use-editable.ts#L408C6-L408C6)

Reply:
gotcha